### PR TITLE
Pass through user selection to next state

### PR DESCRIPTION
**Pull request by hodgestar**
Currently just writes the fields to the contact object

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: test/test_location.js [hunk 1]
```diff
@@ -188,47 +189,128 @@ describe('states.location', function() {
...
                     state:'states:end'
                 })
                 .check(function(api) {
                     var contact = api.contacts.store[0];
                     assert.equal(contact.extra['location:formatted_address'],
-                        'Friend Street, Cape Town 7925, South Africa');
+                            'Friend Street, Amesbury, MA 01913, USA');
                 })
                 .run();
         });
```
Comment (justinvdm) [inline test/test_location.js]: Is this test still relevant? It seems like a duplicate of one of the tests added above?

Comment (hodgestar) [inline test/test_location.js]: Thanks for catching that -- I think I must have somehow duplicated the test while re-organizing things.

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**